etcdserver/v3_server.go

```diff
@@ -562,5 +605,18 @@ func (s *EtcdServer) processInternalRaftRequest(ctx context.Context, r pb.Intern
 	}
 }
 
+func (s *EtcdServer) processInternalRaftRequestWithRetry(ctx context.Context, r pb.InternalRaftRequest) (*applyResult, error) {
```

Comment:
keep this named `processInternalRaftRequest` since the no retry case is rare / never used and rename the old `processInternalRaftRequest` to `processInternalRaftRequestOnce`?